aedttest/aedt_test_runner.py

```diff
@@ -585,14 +591,25 @@ def extract_mesh_or_time_data(
             for setup_name, current_stat in variation_data.items():
                 assert key_name in ["mesh", "simulation_time"]
                 extract = "mesh_name" if key_name == "mesh" else "profile_name"
-
-                new_path = copy_path_to(design_data[extract][variation_name][setup_name], str(self.reference_profiles))
-                design_data[extract][variation_name][setup_name] = new_path
-
+                absolute_path = design_data[extract][variation_name][setup_name]
+
+                # Check if profile exists already to avoid duplicates
```

Comment:
How can paths duplicate if we create unique profile names on export?

Reply:
If I am not wrong the 'unique name' is created in the AEDT project folder ( i.e D:\test\results_2022_06_30_12_03_37\**aedtsolvedproject**\_****.prof)

The folder **aedtsolvedproject** is removed when the Json file is populated, so we are transfering the .prof files to:
D:\test\results_2022_06_30_12_03_37\reference_folder\profiles\_****.prof)

If you have more than one **aedtsolvedproject**, I think the 'unique name' is not valid anymore, because it was created for this specific path, but once we transfer the files to the new folder (reference_folder\profiles) with all the profiles, we could have repeated names.

Another solution could be add another path level like:
D:\test\results_2022_06_30_12_03_37\reference_folder\profiles\aedtsolvedproject\_***prof)

Then not repeated names in the same folder could happen... But at first, I preferred to have all the files at the same level, I could implement this solution if you prefer.